### PR TITLE
Add minutes for coordination meeting #276 (2024-10-10)

### PR DESCRIPTION
**Pull request by jouvin**

### CODE REVIEW

**Comment by jouvin** [comment]
I asked Data Analysis conveners if they wanted to add something about their activity before we publish these minutes... Let's wait up to tomorrow (Friday)...

File: organization/_posts/2024/2024-10-10-coordination.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+The actual [guidelines for projects](https://hepsoftwarefoundation.org/projects/guidelines.html) are on a separate page.
+
+- Do people know of projects (theirs or others) who would like to trial the process of becoming an affiliate?
+
+## Activity Updates
+
+### Data Analysis
+
+- Next meeting on Oct 28: https://indico.cern.ch/event/1461088/
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-10-10-coordination.md:89]: Same minor issue - the links need to be embedded in `<>`s

File: organization/_posts/2024/2024-10-10-coordination.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  - focused on Scipp (https://scipp.github.io/) and ragged data.
```
Comment (graeme-a-stewart) [inline organization/_posts/2024/2024-10-10-coordination.md:90]: Ditto

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Minor link formatting update

**Comment by graeme-a-stewart** [comment]
I'm going to accept this one and just patch in the minor changes, because it would be useful to have the minutes published for an email that needs sent.

**Review by graeme-a-stewart — APPROVED** [review]

**Pull request merged**